src/app/shared/available-logs/available-logs.component.ts

```diff
@@ -26,10 +28,11 @@ export class AvailableLogsComponent extends Base implements OnInit {
     private availableLogsQuery: AvailableLogsQuery,
     private checkerWorkflowQuery: CheckerWorkflowQuery,
     private availableLogsService: AvailableLogsService,
-    @Inject(MAT_DIALOG_DATA) public data: Tag | WorkflowVersion | null
+    @Inject(MAT_DIALOG_DATA) public data: any
```

Comment:
Is this necessary?

Reply:
Had to remind myself why I did that. So before the only data being sent was a version or null, but now the data sent is the version and verified platforms shown here at the bottom.
https://github.com/dockstore/dockstore-ui2/blob/ffcb28d850b515c3b011cb19069e2424dd81e803/src/app/shared/entry/verified-by/verified-by.component.ts
Wasn't sure how to express that.

Reply:
You can define an interface, e.g., syntax not verified, not sure if either is optional or types are right, may not be best name, etc., but something like:

```
interface VersionDialogData {
  version: string;
  verifiedByPlatform: string;
}
```